presto-main/src/test/java/com/facebook/presto/cost/TestFilterStatsRule.java

```diff
@@ -47,7 +47,7 @@ public void tearDownClass()
     }
 
     @Test
-    public void testEstimatableFilter()
+    public void testEstimableFilter()
```

Comment:
My goal is to generally improve codebases/projects, so I try not to get stuck on any individual change.

I've gotten to the point where I'm mostly just letting Google Sheets make suggestions and unless I know the suggestion is wrong, I don't fight it.

I generally spend my time on things where the correction isn't obvious (whether it's because I don't understand the sentence, or because I need to read blocks of code to discern the meaning of a variable/function).

I'm quite happy to drop any suggestions (as individual commits, drops are trivial) and add the words people want to keep into a dictionary / expect file so that my tooling won't flag the next time.

fwiw, for this word, there are ~30k hits and the first hit is:
https://english.stackexchange.com/questions/146616/something-can-be-estimated-therefore-it-is-estimatable

Reply:
I suspected.  It's fine to leave in since it makes doing changes like this easier.